### ISSUE TITLE
view.getState() does not work in debugger

### ISSUE THREAD

**Issue by nyurik** [vega/vega#1025]
Docs:  https://vega.github.io/vega/docs/api/debugging/#state

The `view.getState` throws this error (same in the editor, but the editor is annoyingly minimizing everything, and the code is slightly different).  The code for it is here:

```js
// as in editor
function signalTest(name, op) {
  return !(name === 'parent' || op instanceof vegaDataflow.transforms.Proxy);
}
 // in my code
function signalTest(name, op) {
  return !(name === 'parent' || op instanceof transforms.Proxy);
}
```
in both cases, `Proxy` is not defined.

```
vega.js:35158 Uncaught TypeError: Right-hand side of 'instanceof' is not an object
    at Object.signalTest [as signals] (vega.js:35158)
    at vega.js:34853
    at Array.forEach (<anonymous>)
    at Context.getState (vega.js:34851)
    at View.getState$1 [as getState] (vega.js:35144)
    at <anonymous>:1:17
signalTest @ vega.js:35158
(anonymous) @ vega.js:34853
getState @ vega.js:34851
getState$1 @ vega.js:35144
(anonymous) @ VM5483:1
```

**Comment by jheer** [comment]
Thanks! Proxy should be lower case, will fix soon.

**Comment by nyurik** [comment]
@jheer yep, submitted a PR ^^

**Comment by jheer** [comment]
Fixed in vega-view v2.0.3 / vega v3.0.5.
